README.md

```diff
@@ -63,6 +63,19 @@ fs.writeFileSync('dist/foo.d.ts', await compileFromFile('src/foo.json'))
 
 See [/example](example) for a fully working demo.
 
+### CLI
+Simple CLI utility is provided within the package.
+```
+cat foo.json | json2ts > foo.d.ts
+# or
+json2ts foo.json foo.d.ts
```

Comment:
Can we default `json2ts foo.json` to output foo.d.ts in the same directory as foo.json?